src/main/resources/page/js/dummy-content-displayer.js

```diff
@@ -0,0 +1,7 @@
+class DummyPageContentPopulator {  
+    constructor() {}
+
+    static dateOrTimeDisplayer(id, data) {
```

Comment:
As this is a method, the name of the method should/must/need (to) start with a verb.